tests/smart-cost.spec.js

```diff
@@ -38,7 +38,7 @@ test.describe("smart cost limit", async () => {
     await page.getByTestId("loadpoint-settings-modal").getByLabel("Close").click();
     await expect(page.getByTestId("loadpoint-settings-modal")).not.toBeVisible();
     await expect(page.getByTestId("vehicle-status-charger")).toHaveText("Charging…");
-    await expect(page.getByTestId("vehicle-status-smartcost")).toHaveText("40.0 ct ≤ 40.0 ct");
+    await expect(page.getByTestId("vehicle-status-smartcost")).toHaveText(/(20\.0 ct ≤ 40\.0 ct|40\.0 ct ≤ 40\.0 ct)/);
```

Comment:
Suggestion: shorter version

```suggestion
    await expect(page.getByTestId("vehicle-status-smartcost")).toHaveText(/[24]0\.0 ct ≤ 40\.0 ct/);
```